jobserver/jobs/hourly/dump_db.py

```diff
@@ -0,0 +1,37 @@
+import pathlib
+import subprocess
+import sys
+
+from django_extensions.management.jobs import HourlyJob
+from environs import Env
+
+
+env = Env()
+
+
+class Job(HourlyJob):
+    help = "Dump the database to storage for copying to local dev environments"  # noqa: A003
+
+    def execute(self):
+        database_url = env.str("DATABASE_URL")
+        output = pathlib.Path("/storage/jobserver.dump")
+
+        if not output.exists():
+            print(f"Unknown output path: {output}", file=sys.stderr)
+            sys.exit(1)
+
+        try:
+            with output.open() as f:
+                subprocess.check_call(
+                    [
+                        "pg_dump",
+                        "-Fc",
+                        "--no-acl",
+                        "--no-owner",
+                        database_url,
+                    ],
+                    stdout=f,
+                )
+        except Exception as e:
+            print(str(e), file=sys.stderr)
+            sys.exit(1)
```

Comment:
FWIW, this could be combined to just: 

`sys.exit(exc)`. 

https://docs.python.org/3/library/sys.html#sys.exit, 2nd paragraph.

Reply:
Oh wow, TIL, useful!